ibm/mas_devops/roles/sls_install/tasks/bootstrap.yml

```diff
@@ -9,6 +9,6 @@
         name: "{{sls_instance_name}}-bootstrap"
         namespace: "{{ sls_namespace }}"
       stringData:
-        licensingId: "{{ (bootstrap.license_id is defined or bootstrap.license_id is defined == '') | ternary(bootstrap.license_id, omit ) }}"
-        licensingKey: "{{  (bootstrap.entitlement_file is defined or bootstrap.entitlement_file is defined == '') | ternary(lookup('file', bootstrap.entitlement_file), omit) }}"
-        registrationKey: "{{ (bootstrap.registration_key is defined  or bootstrap.registration_key == '') | ternary(bootstrap.registration_key, omit) }}"
+        licensingId: "{{ (bootstrap.license_id is defined and bootstrap.license_id is defined != '') | ternary(bootstrap.license_id, omit ) }}"
```

Comment:
While we're fiixing this  ... shouldn't this be:

```
(bootstrap.license_id is defined and bootstrap.license_id != '')
```

There's a rogue `is defined` in the second clause?

Same with the `bootstrap.entitlement_file` clause as well